server/graphql_schemas/utils/helpers.py

```diff
@@ -76,6 +76,34 @@ def raise_calendar_error(user_profile):
                                  body=payload).execute()
 
 
+def validate_event_form_fields(input):
```

Comment:
Great job on this implementation @ipheghe. But in the spirit of GraphQL best practices, I think we should perform this validation at the Schema level and not at the local mutation/execution level. This will ensure that all mutations and queries created hereafter obey the same rules and data in our database has integrity. I am suggesting we create simple custom scalars that enforces the different validation rules. See here: https://docs.graphene-python.org/en/latest/types/scalars/